CHANGELOG.md

```diff
@@ -1,3 +1,18 @@
+## 10.10.2 (YYYY-MM-DD)
+
+### Fixed
+* Fixed various corruption bugs when encryption is used. (Realm Core issue [#5360](https://github.com/realm/realm-core/issues/5360))
```

Comment:
```suggestion
* Fixed various corruption bugs when encryption is used. (Realm Core issue [#5360](https://github.com/realm/realm-core/issues/5360), since 10.10.0)
```